synchronicity/synchronizer.py

```diff
@@ -465,7 +465,7 @@ def _wrap_class(self, cls, interface, name):
             for base in cls.__bases__
         )
         new_dict = {self._original_attr: cls}
-        if cls is not None:
+        if cls.__dict__.get("__init__") is not None:
```

Comment:
i think the `is not None` is superfluous here, but yeah this looks like a typo!!